src/txn/txn.c

```diff
@@ -1771,6 +1865,14 @@ __wt_txn_commit(WT_SESSION_IMPL *session, const char *cfg[])
             upd = op->u.op_upd;
```

Comment:
This should happen after we resolve the op, if we do it before then any work done in `resolve_uncommitted_op` is moot.

Reply:
Fixed, moved assignment after the resolution.